eth/chain/rpc_connection.go

```diff
@@ -74,7 +74,7 @@ func (c *RPCConnection) GetLatestBlockNumber() (*uint64, error) {
 	if err != nil {
 		return nil, err
 	}
-	return ref.Uint64(blockNumber), nil
+	return ref.Uint64(blockNumber - 10), nil
```

Comment:
This could use a comment 😄

Reply:
good point! Follow-up PR: https://github.com/worldcoin/hubble-commander/pull/647